### DISCUSSION TITLE
Is it possible to modify the max_client_conn for pgbouncer?

### DISCUSSION THREAD

**Discussion by nacho-carnicero** [supabase/supabase#3928] [Questions]
Hi, 

I am facing sometimes problems in the app we are developing at LabelFlow where we hit the maximum number of connections allowed to pgbouncer. I haven't seen anywhere in the docs how to set this parameter, so I was wondering if it was possible to set it? Or its value is hardcoded (and in case it is which value is it)?

Do you have any recommendation on how to prevent this kind of situation?

Thank you!

**Comment by awalias** [comment]
hey @nacho-carnicero ! I don't think we expose this in the dashboard yet, I will make an issue for it internally

In the meantime can you email support@supabase.com and let them know which project you need tweaking, they may be able to help you here
